certbot/tests/log_test.py

```diff
@@ -27,12 +26,33 @@ def _call(cls, *args, **kwargs):
         return pre_arg_parse_setup(*args, **kwargs)
 
     def test_it(self):
-        with mock.patch('certbot.log.except_hook') as mock_except_hook:
-            with mock.patch('certbot.log.sys') as mock_sys:
-                self._call()
-
+        with mock.patch('certbot.log.util.atexit_register') as mock_register:
```

Comment:
Given that there's now four of these, decorate them? Also oh my god because we use the value of `mock_sys` outside of the with, what?